src/main/java/org/envirocar/wps/StatsForPOI.java

```diff
@@ -220,7 +223,7 @@ public void simpleAlgorithm(Geometry pointOfInterest, double bufferSize, String
 				numberOfTracks = trackIDs.size();
 				String trackDate = "";
 				int trackTime = 0;
-				System.out.println("trackIDSize "+ numberOfTracks);
+				LOGGER.debug("trackIDSize "+ numberOfTracks);

```

Comment:
Not good style because it concatenates the string always, even if debug is not enabled. Use `LOGGER.debug("trackIDSize {}", numberOfTracks);`